### ISSUE TITLE
Support MacOS < 11

### ISSUE THREAD

**Issue by matiEU07** [rnayabed/rangoli#4]
I'm currently on macOS Mojave, and I am sadly not able to run the app

**Comment by matiEU07** [comment]
when force running the application by editing the plist and doing it with terminaI I get:

QQmlApplicationEngine failed to load component
qrc:/Rangoli/Main.qml:14:1: Cannot load library /Users/user/Downloads/rangoli-portable-macos/rangoli.app/Contents/Resources/qml/QtQml/WorkerScript/libworkerscriptplugin.dylib: (dlopen(/Users/user/Downloads/rangoli-portable-macos/rangoli.app/Contents/Resources/qml/QtQml/WorkerScript/libworkerscriptplugin.dylib, 133): Library not loaded: @loader_path/../Frameworks/QtQmlWorkerScript.framework/Versions/A/QtQmlWorkerScript
  Referenced from: /Users/user/Downloads/rangoli-portable-macos/rangoli.app/Contents/Resources/qml/QtQml/WorkerScript/libworkerscriptplugin.dylib
  Reason: image not found)

**Comment by rnayabed** [comment]
Currently Rangoli builds on GitHub actions, which has [deprecated MacOS 10.15 runners](https://github.com/actions/runner-images/issues/5583). 

This means I will have to manually build it, not a big deal. Consider it done!

**Comment by matiEU07** [comment]
Thank you! just to clarify since you mentioned 10.15, the macOS Mojave is 10.14. Is there a chance that it would work?

(thank you for making such awesome software btw!)

**Comment by rnayabed** [comment]
Yes. I am using [this](https://github.com/kholia/OSX-KVM), which supports High Sierra (10.13) onwards.
